community-membership.md

```diff
@@ -130,6 +130,15 @@ The following apply to the subproject for which one would be an owner:
 - Must have been active for 3 months or more for the given sub-project
 - Inactivity for more than 3 months leads to a removal vote by other maintainers and not an automatic removal
 
+### Acceptance
+
+New maintainers can be added to the project by a super-majority (two-thirds / 66.66%) vote of the existing maintainers of the relevant repository. A potential maintainer may be nominated by an existing maintainer from the relevant repository. A vote is conducted in private between the current maintainers over the course of a one week voting period. At the end of the week, votes are counted and a pull request is made on the repo adding the new maintainer to the CODEOWNERS file.
```

Comment:
Will the vote only occur between maintainers of the relevant repo now? That makes more sense.

But additional all maintainers can be notified of the maintainer nomination so that they can reply with objections within the course of a week.

I think the voting email thread (among the maintainers of the repo in question) and the thread to receive objections should be separate.

Reply:
> Will the vote only occur between maintainers of the relevant repo now? That makes more sense.

Yes, language was changed to make that clear.

> But additional all maintainers can be notified of the maintainer nomination so that they can reply with objections within the course of a week.

> I think the voting email thread (among the maintainers of the repo in question) and the thread to receive objections should be separate.

Added